### PR TITLE
chore(google maps): documentation

### PR DESCRIPTION
**Pull request by herefishyfish**
A bit of a start on the docs for Google Maps. 

I'm not sure how/if the docs website renders relative links.

### CODE REVIEW

**Comment by rigor789** [comment]
> I'm not sure how/if the docs website renders relative links.

Not at the moment, will see if that's worth implementing... Might end up flattening into a single README for now instead.

File: packages/google-maps/README.md [hunk 1]
```diff
@@ -155,22 +205,400 @@ The following events are available:
...
+[addCircle](#adding-circles)(circle: [CircleOptions](#circle-options)): [Circle](#circle) | Adds a circle to the map
+[removeCircle](#removing-circles)(circle: [Circle](#circle)) | Removes a circle from the map
+addGroundOverlay(options: GroundOverlayOptions): GroundOverlay | Adds a ground overlay to the map
+removeGroundOverlay(groundOverlay: GroundOverlay) | Removes a ground overlay from the map
+[addPolygon](#adding-polygons)(options: [PolygonOptions](#polygon-options)): [Polygon](#polygons) | Adds a polygon to the map
+[removePolygon](#removing-polygons)(polygon: [Polygon](#polygons)) | Removes a polygon from the map
+[addPolyline](#adding-polylines)(options: [PolylineOptions](#polyline-options)): [Polyline](#polyline) | Adds a polyline to the map
+[removePolyline](#removing-polylines)(polyline: [Polyline]()#polyline) | Removes a polyline from the map
+[animateCamera](#controlling-the-camera)(update: CameraUpdate) | Animates camera to a new position
+snapshot(): Promise<ImageSource> | Returns a platform specific image of the maps current viewport
```
Comment (rigor789) [inline packages/google-maps/README.md]:
```suggestion
snapshot(): Promise\<ImageSource\> | Returns a platform specific image of the maps current viewport
```

File: packages/google-maps/README.md [hunk 2]
```diff
@@ -155,22 +205,400 @@ The following events are available:
...
+`center` | Coordinate |
+`fillColor` | Color \| string |
+`radius` | number |
+`strokeColor` | Color \| string |
+`strokePattern` | PatternItem & Partial\<NativeObject\>[] |
+`strokeWidth` | number |
+`tappable` | boolean |
+`visible` | boolean |
+`zIndex` | number |
+`userData` | { [key: string]: any } |
```
Comment (rigor789) [inline packages/google-maps/README.md]:
```suggestion
`userData` | `{ [key: string]: any }` |
```

File: packages/google-maps/README.md [hunk 3]
```diff
@@ -155,22 +205,400 @@ The following events are available:
...
+| `tappable` | boolean |
+| `strokeWidth` | number |
+| `strokeColor` | Color \| string |
+| `fillColor` | Color \| string |
+| `strokePattern` | PatternItem & Partial\<NativeObject\>[] |
+| `zIndex` | number |
+| `geodesic` | boolean |
+| `strokeJointType` | JointType |
+| `visible` | boolean |
+| `userData` | { [key: string]: any } |
```
Comment (rigor789) [inline packages/google-maps/README.md]:
```suggestion
| `userData` | `{ [key: string]: any }` |
```

File: packages/google-maps/README.md [hunk 4]
```diff
@@ -155,22 +205,400 @@ The following events are available:
...
+| `tappable` | boolean | 
+| `geodesic` | boolean | 
+| `visible` | boolean | 
+| `zIndex` | number | 
+| `jointType` | JointType | 
+| `pattern` | PatternItem & Partial\<NativeObject\>[] | 
+| `color` | Color \| string | 
+| `startCap` | Cap & Partial\<NativeObject\> | 
+| `endCap` | Cap & Partial\<NativeObject\> | 
+| `userData` | { [key: string]: any } |
```
Comment (rigor789) [inline packages/google-maps/README.md]:
```suggestion
| `userData` | `{ [key: string]: any }` | 
```

**Review by rigor789 — COMMENTED** [review]
Some small things that prevent it from rendering within the docs page

**Comment by herefishyfish** [comment]
Applied those fixes thanks @rigor789

**Pull request merged**